packages/autorest.typescript/test/rlcIntegration/generated/bodyComplexRest/src/bodyComplexRestClient.ts

```diff
@@ -7,7 +7,7 @@ import { BodyComplexRestClient } from "./clientDefinitions";
 
 /**
  * Initialize a new instance of `BodyComplexRestClient`
- * @param options type: ClientOptions, the parameter for all optional parameters
+ * @param options - the parameter for all optional parameters
```

Comment:
@qiaozha @joheredi Do you think we could add the `@typescript-eslint/naming-convention` validation in our build?

Reply:
at least not for rest level client, they will easily break this rule .